src/engine/guidance/post_processing.cpp

```diff
@@ -31,16 +32,18 @@ namespace guidance
 
 namespace
 {
-const constexpr double MAX_COLLAPSE_DISTANCE = 25;
+const constexpr double MAX_COLLAPSE_DISTANCE = 30;
```

Comment:
Well. Thats the thing with arbitrary constants. Some time you find out that your first guess wasn't large enough.